pkgs/top-level/all-packages.nix

```
@@ -15203,7 +15203,14 @@ with pkgs;
     gmp-static = gmp.override { withStatic = true; };
   };
 
-  bun = callPackage ../development/web/bun { };
+  inherit
+    ({
+      bun = callPackage ../development/web/bun { };
+      bunBaseline = callPackage ../development/web/bun/baseline.nix { };
+    })
+    bun
+    bunBaseline
+    ;
```

oh and in addition to ^ revert this as well

---

for future reference, the diff you wrote here is also equivalent to:
```suggestion
  bun = callPackage ../development/web/bun { };
  bunBaseline = callPackage ../development/web/bun/baseline.nix { };
```

---

i've done it to bypass [this](https://github.com/NixOS/nixpkgs/blob/master/.github/workflows/check-by-name.yml) check.  without this, workflow just [failes](https://github.com/NixOS/nixpkgs/actions/runs/9196795760/job/25295724717). or this error can be ignored?

---

ah yeah, in that case, you should either:
1. move `bun/baseline.nix` to `by-name/bu/bunBaseline/package.nix` as suggested by the error message
  - you can leave `bun/default.nix` where it's at if you'd like
2. OR implement the `passthru` option as i suggested above
  - i'm realizing this would also probably mean setting `bunBaseline = pkgs.bun.passthru.baseline` in `aliases.nix` instead of modifying `all-packages.nix`

@06kellyjac seems to prefer option 1, i personally prefer option 2, i don't think either of us will care which gets implemented so the decision is up to you :)

---

I was just about to say to fix this issue do a passthru.